app/uart.c

```diff
@@ -369,6 +376,31 @@ bool UART_IsCommandAvailable(void)
 		if (gUART_WriteIndex == DmaLength)
 			return false;
 
+#ifdef ENABLE_MESSENGER
+#ifdef ENABLE_MESSENGER_UART
+
+    if (UART_DMA_Buffer[gUART_WriteIndex] == 'S' && UART_DMA_Buffer[gUART_WriteIndex + 1] == 'M' && UART_DMA_Buffer[gUART_WriteIndex + 2] == 'S' && UART_DMA_Buffer[gUART_WriteIndex + 3] == ':')
```

Comment:
Can we use `memcmp()` here for cleaner code?

Reply:
Was also wondering if we need to require data to start with `SMS`, instead of sending any serial data that comes in?
So it could be a bit more universal. Are there any implications of this?

Reply:
Ohh nevermind  I guess reading from CHIRP would then trigger message send. Perhaps in the future it should be limited to only when messenger screen is active.

Reply:
maybe `if (strncmp(((char*)UART_DMA_Buffer) + gUART_WriteIndex, "SMS:",4) == 0)` ?